io.github.tomluchowski.OpenDungeonsPlus.yml

```diff
@@ -0,0 +1,151 @@
+app-id: io.github.tomluchowski.OpenDungeonsPlus
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: /app/games/opendungeons-plus
```

Comment:
command should just be a command and be in the path, like `/app/bin`. NOT an absolute path

Reply:
so if not command: /app/games/opendungeons-plus -- then what ? Please give hint.

Reply:
`opendungeons-plus`

Reply:
But the binary should be in `/app/bin`, not `/app/games`.

Reply:
ahh as usual all the roads leads to CMakeList.txt:

> # Set the data path depending on the platform
> if(WIN32)
>     set(OD_DATA_PATH ".")
>     set(OD_PLUGINS_CFG_PATH ".")
>     set(OD_BIN_PATH ${CMAKE_INSTALL_PREFIX})
> else()
>     # Set binary and data install locations if we want to use the installer
>     set(OD_BIN_PATH ${CMAKE_INSTALL_PREFIX}/games CACHE PATH "Absolute path to the game binary directory")
>     set(OD_DATA_PATH ${CMAKE_INSTALL_PREFIX}/share/games/${PROJECT_NAME} CACHE PATH "Absolute path to the game data directory") 
>     set(OD_SHARE_PATH ${CMAKE_INSTALL_PREFIX}/share CACHE PATH "Absolute path to the shared data directory (desktop file, icons, etc.)")
>     set(OD_MAN_PATH ${OD_SHARE_PATH}/man CACHE PATH "Absolute path to the manpages directory")
>     # Set the plugins.cfg file path to a common but architecture-dependent location.
>     # Because the plugins.cfg Ogre plugins path path may vary depending on the architecture used.
>     set(OD_PLUGINS_CFG_PATH /etc/${PROJECT_NAME} CACHE PATH "Absolute path to the Ogre plugins.cfg file")
>     
> endif()
so I do modify it so nothing is broken ?

Reply:
Uhh seems that setting OD_BIN_PATH by proper cmake command did the job.